help/en/html/apps/DecoderPro/FAQ.shtml

```diff
@@ -85,38 +81,38 @@
           index each time you update.
         </dd>
 
-        <dt><a name="update" id="update">How can I get
```

Comment:
If you want the JavaHelp system to index and provide links to these, there's a specific format for anchors needed. It needs to have both "name" and "id".  See https://www.jmri.org/help/en/html/doc/Technical/Help.shtml

Reply:
Bob - 

Actually, I'm the one who added the text to Help.sthml that says that!  However, when I ran this page through validator.w3c.org under HTML 4.01, it told me that there was no such thing as "name" so I took them out thinking it had something to do with being embedded within the dt tag.

Jerry

Reply:
P.S. and they weren't there in the existing PanelPro FAQ file so I didn't add them.